Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/ProductionPerformanceContext.swift

```diff
@@ -0,0 +1,59 @@
+// swiftlint:disable force_try
+import MongoSwift
+import StitchCore
+import StitchCoreSDK
+import StitchCoreAdminClient
+import StitchDarwinCoreTestUtils
+@testable import StitchCoreRemoteMongoDBService
+import StitchCoreLocalMongoDBService
+@testable import StitchRemoteMongoDBService
+
+class ProductionPerformanceContext: SyncPerformanceContext {
+    let dbName: String
+    let collName: String
+    let userId: String
+    let client: StitchAppClient
+    let mongoClient: RemoteMongoClient
+    let coll: RemoteMongoCollection<Document>
+    let testParams: TestParams
+    let transport: FoundationInstrumentedHTTPTransport
+    let harness: SyncPerformanceIntTestHarness
+
+    let joiner = ThrowingCallbackJoiner()
+    let streamJoiner = StreamJoiner()
+
+    required init(harness: SyncPerformanceIntTestHarness,
+                  testParams: TestParams,
+                  transport: FoundationInstrumentedHTTPTransport) throws {
+        self.harness = harness
+        self.transport = transport
+        self.testParams = testParams
+
+        dbName = harness.stitchTestDbName
+        collName = harness.stitchTestCollName
+        client = harness.outputClient
+        mongoClient = harness.outputMongoClient
+        coll = mongoClient.db(dbName).collection(collName)
+        userId = client.auth.currentUser?.id ?? "No User"
+
+        coll.sync.proxy.dataSynchronizer.isSyncThreadEnabled = false
+        coll.sync.proxy.dataSynchronizer.stop() // Failing here occaisonally
```

Comment:
[q] Is this still failing here occasionally? If so, we should fix it.

Reply:
Ill remove this comment, I havent seen it fail in the past few days

Reply:
I must have fixed it at some point while fixing something else :)